src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -116,11 +112,12 @@
             font-size: 80%;
 
             .asset-filename {
-                max-width: 175px;
+                max-width: 110px;
                 overflow: hidden;
                 text-overflow: ellipsis;
                 display: inline-block;
                 white-space: nowrap;
+                text-align: end;
```

Comment:
what does this do? can I see a before/after for comparison

Reply:
Actually, nothing. Will remove. Thanks.